app/components/Authentication/Radius/Radius.jsx

```diff
@@ -0,0 +1,529 @@
+                            fullWidth={false}
+                            autoFocus
+                            onChange={(e) => {
+                                this.setState({ newUserId: e.target.value });
+                            }}
+                        />
+                        <Subheader>Assigned Policies</Subheader>
+                        <PolicyPicker
+                            height="120px"
+                            selectedPolicies={this.state.newUserObject.policies}
+                            onSelectedChange={(policies) => {
+                                let user = this.state.newUserObject;
+                                user.policies = policies;
+                                this.setState({ newUserObject: user });
+                            }}
+                        />
+                    </List>
+                </Dialog>
+            );
+        }
+
+        let renderDeleteConfirmationDialog = () => {
+            const actions = [
+                <FlatButton label="Cancel" primary={true} onTouchTap={() => this.setState({ openDeleteModal: false, deleteUserId: '' })} />,
+                <FlatButton label="Delete" style={{ color: white }} hoverColor={red300} backgroundColor={red500} primary={true} onTouchTap={() => this.DeleteUser(this.state.deleteUserId)} />
+            ];
+
+            return (
+                <Dialog
+                    title={`Delete Confirmation`}
+                    modal={false}
+                    actions={actions}
+                    open={this.state.openDeleteModal}
+                >
+
+                    <p>You are about to permanently delete user {this.state.deleteUserId}.  Are you sure?</p>
+                    <em>To disable this prompt, visit the settings page.</em>
+                </Dialog>
+            )
+        }
+
+
+        return (
+            <div>
+                {this.state.openEditUserDialog && renderEditUserDialog()}
+                {this.state.openNewUserDialog && renderNewUserDialog()}
+                {this.state.openDeleteModal && renderDeleteConfirmationDialog()}
+                <Tabs>
+                    <Tab label="Manage users" >
+                        <Paper className={sharedStyles.TabInfoSection} zDepth={0}>
+                            Here you can add, edit or delete users registred with this backend
+                        </Paper>
+                        <Paper className={sharedStyles.TabContentSection} zDepth={0}>
+                            <Toolbar>
+                                <ToolbarGroup firstChild={true}>
+                                    <FlatButton
+                                        primary={true}
+                                        label="NEW USER"
+                                        disabled={this.state.newSecretBtnDisabled}
+                                        backgroundColor={green500}
+                                        hoverColor={green400}
```

Comment:
Don't think so, I believe they were carried forward from the first implementation of the generic secret browser.

Plus I'm not exactly a terrific designer so I struggle with choosing proper color palettes.
Maybe it's worth removing all hardcoded colors like in this case and instead change the global material-ui palette.